admin/content.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+require 'functions.php';
+
+$db = connectDatabase();
+$stmt = $db->query("SELECT `name`,`sub_title`,`tagline`FROM `content`;");
+$content_data = $stmt->fetch();
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Content</title>
+</head>
+<body>
+
+<form method="POST" action="submit_content.php">
+    Name: <input type="text" name="name" value="<?php echo $content_data["name"] ?>"><br/>
```

Comment:
Lines 20 to 22 consider using labels instead of free text

Reply:
changed to p's